src/apps/demo-modal/demo-modal.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from "react";
+import { useDispatch } from "react-redux";
+import ModalWrapper from "../../components/modal/modal-wrapper";
+import { openModal } from "../../core/modal.slice";
+
+export interface DemoModalProps {
+  ariaLabelModalOne: string;
+  ariaLabelModalTwo: string;
+  screenReaderModalDescriptionText: string;
+}
+
+const DemoModal: React.FC<DemoModalProps> = ({
+  ariaLabelModalOne,
+  ariaLabelModalTwo,
+  screenReaderModalDescriptionText
```

Comment:
I think all the texts should be handled in the way of the newly introduced text handling feature:
https://github.com/danskernesdigitalebibliotek/dpl-react/blob/main/docs/ui_text_handling.md#ui-text-handling